src/common/Types.ts

```diff
@@ -424,6 +424,8 @@ export interface ButtonProps extends CommonStyledProps<ButtonStyleRuleSet>, Comm
     onFocus?: (e: FocusEvent) => void;
     onBlur?: (e: FocusEvent) => void;
 
+
+    onAccessibilityTap?: Function; // iOS-only prop, when a button is double tapped in accessibility mode we have a chance to act on it
```

Comment:
For IOS-specific props, we append IOS on the end (see prop right below this one).